specs/applicationdata/ApplicationData.md

```diff
@@ -0,0 +1,428 @@
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.temporaryfolder
+        String TemporaryPath { get; };
+
+        /// Return a StorageFolder for the local cache data store not included in backup and restore operations.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.localcachefolder
+        Windows.Storage.StorageFolder LocalCacheFolder { get; };
+
+        /// Return a StorageFolder for the local data store. This location is backed up to the cloud.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.localfolder
+        Windows.Storage.StorageFolder LocalFolder { get; };
+
+        /// Return a StorageFolder for the machine data store.
+        /// @note This location is per-machine (not per-user like other *Folder properties).
+        Windows.Storage.StorageFolder MachineFolder { get; };
+
+        /// Return a StorageFolder for the roaming data store.
+        /// @warning Roaming data and settings are no longer supported after Windows 10 version 1901 (aka 19H1 aka 10.0.18363.0).
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.roamingfolder
+        Windows.Storage.StorageFolder RoamingFolder { get; };
+
+        /// Return a StorageFolder for the shared data store.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.sharedlocalfolder
+        Windows.Storage.StorageFolder SharedLocalFolder { get; };
+
+        /// Return a StorageFolder for the temporary data store.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.temporaryfolder
+        Windows.Storage.StorageFolder TemporaryFolder { get; };
+
+        /// Return the settings container in the local data store.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.localsettings
+        ApplicationDataContainer LocalSettings { get; };
+
+        /// Return the settings container in the roaming data store.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.roamingsettings
+        ApplicationDataContainer RoamingSettings { get; };
+
+        /// Remove all data from the specified data store.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.clearasync
+        Windows.Foundation.IAsyncAction ClearAsync(ApplicationDataLocality locality);
+
+        /// Remove all data from the shared data store in the specified subfolder.
+        /// @see SharedLocalPath
+        /// @see SharedLocalFolder
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.clearpublishercachefolderasync
+        Windows.Foundation.IAsyncAction ClearPublisherCacheFolderAsync(String folderName);
+
+        /// Return the path for the shared data store for the publisher of the app.
```

Comment:
GetPublisherCachePath and GetPublisherCacheFolder comments seem to be wrong, since they say that they ignore the `folderName` parameter and just give you the SharedLocalPath / SharedLocalFolder.